src/SFA.DAS.EmploymentCheck.Functions/Application/Services/EmployerAccount/EmployerAccountService.cs

```diff
@@ -56,135 +60,87 @@ public class EmployerAccountService
-
-                await StoreAccountsResponse(new AccountsResponse(
-                    apprenticeEmploymentCheck.Id,
-                    apprenticeEmploymentCheck.CorrelationId,
-                    apprenticeEmploymentCheck.AccountId,
-                    "NULL", // payeSchemes
-                    response.ToString(),
-                    (short) response.StatusCode));
-                throw
-                    new InvalidOperationException(
-                        nameof(response)); // TODO: Create a custom business exception for this condition
+                accountsResponse.HttpResponse = e.Message;
+                content = (TResponse)new object();
+                _logger.LogError($"EmployerAccountService.Get(): ERROR: {accountsResponse.HttpResponse}");
             }
-
-
-            var json = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
-            if (string.IsNullOrEmpty(json))
+            finally
             {
-                // read of content failed
-                _logger.LogError(
-                    $"\n\n{thisMethodName}: converting the response Content to string returned an Empty/Null string (the status code is [{response.StatusCode}])");
-
-                await StoreAccountsResponse(new AccountsResponse(
-                    apprenticeEmploymentCheck.Id,
-                    apprenticeEmploymentCheck.CorrelationId,
-                    apprenticeEmploymentCheck.AccountId,
-                    "NULL", // payeSchemes
-                    response.ToString(),
-                    (short) response.StatusCode));
-                throw
-                    new InvalidOperationException(
-                        nameof(response)); // TODO: Create a custom business exception for this condition
+                try
+                {
+                    var allEmployerPayeSchemes = new StringBuilder();
+                    foreach (var payeScheme in employerPayeSchemes.PayeSchemes)
+                    {
+                        allEmployerPayeSchemes.Append($", {payeScheme}");
+                    }
+
+                    // trim the comma at start of string
+                    responsePayeSchemes = allEmployerPayeSchemes.ToString();
+                    responsePayeSchemes = responsePayeSchemes.Remove(0, 1);
+
+                    accountsResponse.PayeSchemes = responsePayeSchemes;
+                    accountsResponse.HttpResponse = response != null ? response.ToString() : "ERROR: Get() - The call to the Accounts API returned no response data.";
+                    accountsResponse.HttpStatusCode = (short)response.StatusCode;
+
+                    await _repository.Save(accountsResponse);
+                }
+                catch (Exception e)
+                {
+                    _logger.LogError($"LearnerService.SendIndividualRequest(): ERROR: the AccountsRepository Save() method threw an Exception [{e}]");
```

Comment:
should thsi say `EmployerAccountService.cs`

Reply:
Yes, done.